08LessonEight/homework/exercise-06.html

```diff
@@ -1,10 +1,34 @@
-<!DOCTYPE html>
-<html>
-
-<head>
-    <title>Exercise 06</title>
-</head>
-
-<body></body>
-
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Homework 8</title>
+    <link rel="stylesheet" href="css/exercise-06-style.css"/>
+</head>
+
+<body>
+    <div class="parent">
+        <section>
+            <article>
+              <img src="img/truck.jpg" alt="">
```

Comment:
While we haven't discussed it in class, it's best practice to always include an `alt` text value. It's for accessibility reasons.

"Note: The term "ALT tag" is a common shorthand term used to refer to the ALT attribute within in the IMG tag. Any time you use an image, be sure to include an ALT tag or ALT text within the IMG tag. Doing so will provide a clear text alternative of the image for screen reader users." - http://accessibility.psu.edu/images/imageshtml/